corehq/apps/app_manager/static/app_manager/json/commcare-profile-settings.yaml

```diff
@@ -350,10 +350,10 @@
 - name: "Grid Menus"
   description: "If enabled module and form menus will display in a grid view instead of a list."
   id: "cc-grid-menus"
-  values: ["yes", "no"]
-  value_names: ["Yes", "No"]
-  default: "no"
-  disabled_default: "no"
+  values: ["none", "first", "some", "all"]
+  value_names: ["None", "First Screen", "Some Screens", "All Screens"]
```

Comment:
"Some screens" is rather ambiguous. It looks like there was discussion in the spec about having a hint as to what that means. Was that resolved?

Could add something to the description, or change "Some Screens" to something like "Customize per Module."